packages/components/src/components/SquiggleViewer/SquiggleValueChart.tsx

```diff
@@ -1,17 +1,19 @@
 import { memo } from "react";
 
+import { SqBoxedValue } from "../../../../squiggle-lang/src/public/SqValue/index.js";
 import { SqValueWithContext } from "../../lib/utility.js";
 import { widgetRegistry } from "../../widgets/registry.js";
 import { PlaygroundSettings } from "../PlaygroundSettings.js";
 
 export const SquiggleValueChart = memo<{
   value: SqValueWithContext;
   settings: PlaygroundSettings;
-}>(function SquiggleValueChart({ value, settings }) {
+  boxed?: SqBoxedValue;
```

Comment:
It's a bit awkward that we return the SqBoxedValue, which already contains the ``SqValue``, but other options seemed much messier.

Reply:
Huh. Maybe we could do this instead?
- remove `SqBoxedValue` and `SqBoxed`
- in `wrapValue`, if the value is boxed, unpack it and store its `BoxedArgs` in `SqValue` (or maybe even in `SqValueContext`, but a common field in `SqAbstractValue` base class is probably a better fit); otherwise store the default empty `BoxedArgs`, similar to "force boxes".

Then on the frontend you'd always have tags available.